std_msgs/Header header
float64 alt_ref            # alt_ref reading (from int -999 to int +200 with float 2.5)
float64 alt_ref_variance   # alt_ref_variance reading 0 in interpreted as unknown

